components/zoom_admin/sources/meeting-started/meeting-started.mjs

```diff
@@ -19,11 +19,10 @@ export default {
   },
   async run(event) {
     const { payload } = event;
-    const { object } = payload;
     this.$emit(event, {
-      summary: `Meeting ${object.topic} started`,
-      id: object.uuid,
-      ts: +new Date(object.start_time),
+      summary: `Meeting ${payload?.object.topic} started`,
+      id: payload?.object.uuid,
+      ts: +new Date(payload?.object.start_time),
```

Comment:
Should the check be: `payload.object?.topic` ?

And if `object` not in `payload`, then the event will be:

```js
summary: Meeting undefined started,
id: undefined,
ts: Invalid Date
```